src/main/java/com/eerussianguy/blazemap/api/maps/Layer.java

```diff
@@ -31,7 +31,7 @@
  *
  * @author LordFokas
  */
-public abstract class Layer implements RegistryEntry, Consumer {
+public abstract class Layer implements RegistryEntry, Consumer, IMapPart {
```

Comment:
add `@Override` to the interface methods?

Reply:
- [ ] Add `@Override` to interface methods on `Layer`